erigon-lib/direct/sentry_client_mock.go

```diff
@@ -10,14 +10,14 @@
 package direct
 
 import (
-	"context"
-	"reflect"
-
-	"github.com/ledgerwatch/erigon-lib/gointerfaces/sentry"
-	"github.com/ledgerwatch/erigon-lib/gointerfaces/types"
-	"go.uber.org/mock/gomock"
-	"google.golang.org/grpc"
-	"google.golang.org/protobuf/types/known/emptypb"
+	context "context"
+	reflect "reflect"
+
+	sentry "github.com/ledgerwatch/erigon-lib/gointerfaces/sentry"
+	types "github.com/ledgerwatch/erigon-lib/gointerfaces/types"
+	gomock "go.uber.org/mock/gomock"
+	grpc "google.golang.org/grpc"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
```

Comment:
we shouldn't need this change the package names will default to these values anyway

Reply:
Yes, I agree, but this file is auto-generated, and these names were added automatically by the `make mocks` command, which is used to create the new mocks added with the new test.

This file was probably generated with an older version of the mock tool and will continue to be changed when a new mock is generated using the tool.

I decided to add this change to avoid this happening in the future.

Do we still need to remove it?

Reply:
Ah ha, I hadn't spotted is was auto generated.  Fine to leave it as it is in that case 👍